### DISCUSSION TITLE
GitHub's Changed-File Filter Does Not Consider the File's Whole Path

### DISCUSSION THREAD

**Discussion by ssiano** [community/community#7328] [Enterprise]
GitHub's changed-file filter does not consider the file's whole path, but only the truncated part shown in the browser dropdown list.

This is why the `vtools` filter in the attachment below finds three files, whereas the `devtools` filter in the attachment below only finds two files.

Here, I am filtering for everything in `src/main/java/com/google/devtools/`, and would like to do so by typing `d` `e` `v` `t` . . .  However, if I do this I will miss the file whose dropdown entry starts with `...vtools`.

This is a trivial example, but in large PR's from large repos, the ability to filter based on the files' whole paths is quite helpful.
![vtools](https://user-images.githubusercontent.com/1631734/140437597-fbff42d0-c135-44c5-8bda-e1a40b7ce3fa.png)
![devtools](https://user-images.githubusercontent.com/1631734/140437609-53dd10e5-899d-480f-9ecb-0e63002fe3b0.png)

**Comment by ssiano** [comment]
This is still an issue in GitHub Enterprise Server 3.3.4.

However, this issue will soon be solved:  [Pull Request File Tree (Beta) Feedback](https://github.com/github/feedback/discussions/12341).

The **Pull Request File Tree** consider's the file's whole path when filtering!

**Comment by rungrotArunon** [comment]
Thanks you
